test/distributed/elastic/rendezvous/etcd_rendezvous_backend_test.py

```diff
@@ -24,6 +26,7 @@
 )
 from torch.distributed.elastic.rendezvous.etcd_server import EtcdServer
 from torch.distributed.elastic.rendezvous.etcd_store import EtcdStore
+from torch.distributed.elastic.rendezvous.api import RendezvousStoreInfo
```

Comment:
let's keep all import's in alphabetic order. Move it in between `from torch.distributed.elastic.rendezvous import` and `from torch.distributed.elastic.rendezvous.etcd_rendezvous_backend import`